### DISCUSSION TITLE
How to get each element of the list?

### DISCUSSION THREAD

**Discussion by isaced** [AutomaApp/automa#181] [Q&A] [answered]
For example, get the text content of each element in a list.

I have tried [Browserflow](https://browserflow.app/) and can use [`loop-elements`](https://docs.browserflow.app/reference/browserflow-commands/control-flow/loop-elements). I don’t know how Automa does it?

**Comment by Kholid060** [comment, marked as answer]
Automa doesn't have a feature like `loop-elements`. But if you want to get the text content from multiple elements, you can check the `multiple` checkbox like the image below. It will get all the text content from all the elements that match the selector.

![image](https://user-images.githubusercontent.com/22908993/145762237-b36fac1c-ddad-41d3-a3b3-a5219c7e32d7.png)

Or you can use the javascript code block and write the code like below
```js
const elements = document.querySelectorAll('.selector-here');
const textContent = Array.from(elements).map((element) => element.innerText);

automaNextBlock({ column: textContent })
```

**Reply by techflag** [reply]
Need to click on each element in the data list，how to set the workflow，TKS
